fiftyone/utils/utils3d.py

```
@@ -13,10 +13,14 @@
 
 import numpy as np
 import scipy.spatial as sp
+from scipy.spatial.transform import Rotation as R
+from typing import Any, Dict, List, Tuple, Union
+from dataclasses import dataclass
 
 import eta.core.numutils as etan
 import eta.core.utils as etau
 
+import fiftyone as fo
```

Ruff (0.12.2)

`fiftyone` imported but unused

Remove unused import: `fiftyone`

(F401)